op-acceptance-tests/tests/isthmus/operator_fee/balance_snapshot.go

```diff
@@ -0,0 +1,98 @@
+package isthmus
+
+import (
+	"fmt"
+	"math/big"
+
+	"github.com/ethereum-optimism/optimism/devnet-sdk/testing/systest"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+type BalanceSnapshot struct {
+	BlockNumber         *big.Int
+	BaseFeeVaultBalance *big.Int
+	L1FeeVaultBalance   *big.Int
+	SequencerFeeVault   *big.Int
+	OperatorFeeVault    *big.Int
+	FromBalance         *big.Int
+}
+
+// String returns a formatted string representation of the balance snapshot
+func (bs *BalanceSnapshot) String() string {
+	if bs == nil {
+		return "nil"
+	}
+
+	return fmt.Sprintf(
+		"BalanceSnapshot{Block: %v, BaseFeeVault: %v, L1FeeVault: %v, SequencerFeeVault: %v, "+
+			"OperatorFeeVault: %v, WalletBalance: %v}",
+		bs.BlockNumber,
+		bs.BaseFeeVaultBalance,
+		bs.L1FeeVaultBalance,
+		bs.SequencerFeeVault,
+		bs.OperatorFeeVault,
+		bs.FromBalance,
+	)
+}
+
+// Add adds this snapshot's balances to another snapshot and returns a new snapshot
+// This is typically used to apply changes to a starting balance snapshot
+func (bs *BalanceSnapshot) Add(start *BalanceSnapshot) *BalanceSnapshot {
```

Comment:
Is it worth adding (regular Go) tests for stuff like this?
For example, if a new field were added to the BalanceSnapshot struct, it's easy to forget to add it here as well.

Reply:
Couldn't hurt.

Reply:
Fixed in 4ddba19a657fd0d4c7d7e54bbf25c7330081ec2a